### PR TITLE
Fix BigQuery weekly active visitor history

### PR DESCRIPTION
**Pull request by steve8708**
## Summary
- Repair the known malformed weekly active visitor SQL on the canonical BigQuery dashboard.
- Add a regression test for non-empty SQL with an always-true time-range branch.

## Verification
- Analytics repair spec: 21 passed
- Analytics typecheck passed
- `pnpm guards`: 71 passed
- Beta dashboard panel repaired and rendered with historical Jun-Sep data; corrected BigQuery query returned nonzero WAU values before Sep 1.

### CODE REVIEW

File: templates/analytics/server/lib/canonical-first-party-dashboard-repair.ts [hunk 1]
```diff
@@ -73,6 +73,10 @@ SELECT date, template, visitors
 FROM wau
 ORDER BY date, template`;
 
+function isMalformedFirstPartyBigQueryWauSql(sql: string): boolean {
+  return sql.includes("WHEN '{{timeRange}}' = '{{timeRange}}'");
```
[the remark this answers is not in the document]
Comment (steve8708) [reply templates/analytics/server/lib/canonical-first-party-dashboard-repair.ts]: Tightened the detector to compare normalized SQL against the complete known malformed query, so a customized variant is preserved. Added a regression test that changes the ORDER BY clause while retaining the malformed predicate and verifies no update is written. Focused suite passes 22/22.

File: templates/analytics/server/lib/canonical-first-party-dashboard-repair.ts [hunk 2]
```diff
@@ -73,6 +73,22 @@ SELECT date, template, visitors
...
 ORDER BY date, template`;
 
+const MALFORMED_FIRST_PARTY_BIGQUERY_WAU_SQL =
+  FIRST_PARTY_BIGQUERY_WAU_SQL.replace(
+    "WHEN '{{timeRange}}' = '7d'",
+    "WHEN '{{timeRange}}' = '{{timeRange}}'",
+  );
+
+function normalizeSql(sql: string): string {
+  return sql.replace(/\s+/g, " ").trim();
```
[the remark this answers is not in the document]
Comment (steve8708) [reply templates/analytics/server/lib/canonical-first-party-dashboard-repair.ts]: Switched to an exact trimmed comparison of the complete shipped malformed SQL, so whitespace inside literals or comments cannot match. Added the requested literal-change regression; focused suite passes 23/23.
